README.md

```diff
@@ -121,8 +121,9 @@ app.streamItems(
 
 ## Building
 
-1. `yarn install --pure-lockfile`
-2. `yarn start` to start Webpack in development mode (watches changes), or `yarn build` to create dist files.
+1. `npm install --global lerna`
```

Comment:
No need to globally install lerna - it's already in the package.json's dependencies, and when you run yarn commands it default to this locally-installed version :)